src/main/webapp/css/home.css

```diff
@@ -15,3 +16,20 @@
 .error-message {
   padding: 5% 15%;
 }
+
+.search-bar {
+  background-color: #f7f7f7;
+  padding-bottom: 0.8%;
+  padding-top: 0.3%;
+}
+
+.no-padding {
+  padding: 2% 0px 0px 0px !important;
```

Comment:
> Omit unit specification after “0” values, unless required.

https://google.github.io/styleguide/htmlcssguide.html#0_and_Units

Also, since the left and right padding is the same, the shorthand only needs three values.

https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#:~:text=When%20three%20values%20are%20specified,in%20that%20order%20(clockwise).